WordPress/WordPress.xcodeproj/project.pbxproj

```diff
@@ -3104,27 +3098,6 @@
 			);
 			target = 1D6058900D05DD3D006BFB54 /* WordPress */;
 		};
-		24CE68AF2CD3375300C7B37D /* Exceptions for "Classes" folder in "WordPressShareExtension" target */ = {
-			isa = PBXFileSystemSynchronizedBuildFileExceptionSet;
-			membershipExceptions = (
```

Comment:
`AppConfiguration.swift` was not used in app extensions, so I removed it from these targets. All app extension have access to new `BuildSettings`, so there are never going to need `AppConfiguration.swift`.